mirrors.yml

```diff
@@ -25,6 +22,18 @@ tencent:
 tuna:
     desc: Tsinghua University TUNA Association OSS Mirror
     url: https://mirrors.tuna.tsinghua.edu.cn/anthon/
+tuna4:
+    desc: Tsinghua University TUNA Association OSS Mirror
```

Comment:
There should be different descriptions for the 4 and the 6. For example, `(IPv4 only) (IPv6 only)`.